src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -1093,12 +1111,23 @@ void LevelZeroCompilerInDriver<TableExtension>::getMetadata(TableExtension* grap
     }
 
     if (!isStateInputName(arg.name) && !isStateOutputName(arg.name)) {
+        ze_graph_argument_metadata_t metadata;
+        result = graphDdiTableExt->pfnGraphGetArgumentMetadata(graphHandle, index, &metadata);
```

Comment:
This API call impacts most extension versions (1.2-1.6), the API exists in these versions, can we get right info for all these versions?

Reply:
Good point, actually correct info is only for ext version > 1.6

Reply:
Since the plugin does not support ext version < 1.2 I removed support from adapter as well. It was easier for keeping versioning in place in this case.